sdk/hardwaresecuritymodules/Azure.ResourceManager.HardwareSecurityModules/src/Generated/Models/OutboundEnvironmentEndpointCollection.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Azure.Core;
+
+namespace Azure.ResourceManager.HardwareSecurityModules.Models
+{
+    /// <summary> Collection of OutboundEnvironmentEndpoint. </summary>
+    internal partial class OutboundEnvironmentEndpointCollection
```

Comment:
Avoid a module name ending with `Collection` to differentiate it from resource collection classes. Try to rename it to `OutboundEnvironmentEndpointGroup` or `OutboundEnvironmentEndpointList`